design-documents/storefront/storefront-api.md

```diff
@@ -0,0 +1,103 @@
+# Storefront API Technical Vision
+
+## Definition
+
+*Storefront APIs* - public operations and queries that 
+will be available at the application storefront for buyers.
+
+
+### Desired state
+
+![Storefront APIs](storefront-api/storefront-api-01.png)
+
+
+#### Storefront APIs are separated from admin APIs.
+
+We are using different object representation for storefront and admin scenarios.
+We are applying different rules and restrictions for such objects depending on where these objects appear.
+Currently, we do not have ACL resources specified for the storefront.
```

Comment:
ACL on storefront is possible in case of B2B

Reply:
Even for B2B, this is not an admin ACL, so statement still correct. Storefront operations use another authorization, ACL, etc.

Reply:
The statement is not correct. We have ACL on storefront and we need it(in all contexts, we don't want to expose orders and addresses to different people). However, this is not a reason to stop API separation. Why did you mention ACL in this context?